src/IntSet.java

```diff
@@ -0,0 +1,205 @@
+			this.set=null;
+			System.exit(-1);
+		}
+	}
+	public boolean validateInput(int[] arr)
+	{
+		if(arr.length>range)
+		{
+			return false;
+		}
+		for(int value:arr)
+		{
+			if(value>range||value<1)
+			{
+				return false;
+			}
+		}
+		return !containDuplicateElements(arr);
+	}
+	
+	private boolean containDuplicateElements(int[] arr)
+	{
+		for(int loop_var=0;loop_var<arr.length;loop_var++)
+		{
+			for(int inner_loop_var=loop_var+1;inner_loop_var<arr.length;inner_loop_var++)
+			{
+				if(arr[inner_loop_var]==arr[loop_var])
+				{
+					return true;
+				}
+			}
+		}
+		return false;
+	}
+	
+	public boolean isMember(int x)
+	{
+		for(int value:set)
+		{
+			if(value==x)
+			{
+				return true;
+			}
+		}
+		return false;
+	}
+	
+	public static void printSet(IntSet s)
+	{
+		for(int value:s.set)
+		{
+			System.out.print(value+", ");
+		}
+	}
+
+	public int size()
+	{
+		return set.length;
+	}
+	
+	public boolean isSubSet(IntSet s)
+	{
+		int set_length=this.set.length;
+		int subset_length=s.set.length;
+		if(subset_length>set_length)
+		{
+			return false;
+		}
+		else
+		{
+			for(int loop_var=0;loop_var<subset_length;loop_var++)
+			{
+				boolean flag=false;
+				int to_search=s.set[loop_var];
+				for(int inner_loop_var=0;inner_loop_var<set_length;inner_loop_var++)
+				{
+					if(to_search==this.set[inner_loop_var]) //Invert this
+					{
+						flag=true;
+					}
+				}
+				if(flag==false)
+				{
+					return false;
+				}							// Till here
+			}
+		}
+		return true;
+	}
+
+	public void setUniversalSet(int range)
+	{
+		this.range=range;
+	}
+	public IntSet getComplement()
+	{
+		int loop_var=0;
+		for(loop_var=1;loop_var<=range;loop_var++)
+		{
+			universal_set.add(loop_var);
+		}
+		for(loop_var=0;loop_var<set.length;loop_var++)
+		{
+			if(universal_set.contains(set[loop_var]))
+			{
+				universal_set.remove(universal_set.indexOf(set[loop_var]));
+			}
+		}
+		return new IntSet(toArray(universal_set));
+	}
+
+	public static IntSet union(IntSet s1, IntSet s2)
+	{
+		ArrayList <Integer> result=new ArrayList<Integer>();
+		result=merge(s1.set,result);
+		result=merge(s2.set,result);
+		return new IntSet(toArray(result));
+	}
+	private static ArrayList<Integer> merge(int to_merge[],ArrayList<Integer> merge_into)
+	{
+		int set_length=to_merge.length;
+		for(int loop_var=0;loop_var<set_length;loop_var++)
+		{
+			boolean flag=false;
+			int to_search=to_merge[loop_var];
+			for(int value:merge_into)
+			{
+				if(value==to_search)
+				{
+					flag=true;
+				}
+			}
+			if(flag==false)
+			{
+				merge_into.add(to_search);
+			}
+		}
+		return merge_into;
+	}
+	private static int[] toArray(ArrayList<Integer> input)
+	{
+		Collections.sort(input);
+		int temp=0;
+		int arr[]=new int[input.size()];
+		for(int value:input)
+		{
+			arr[temp++]=value;
+		}
+		return arr;
+	}
+
+}
```

Comment:
1. I don't think you needs to create a method for ArrayList to Array conversion, probably you could use toArray method of  Arraylist for that.
2. You should not use ArrayList in this assignment. You could confirm with this Assignment owner.